Velaptor/Content/Sound.cs

```diff
@@ -153,7 +153,15 @@ public float PlaySpeed
         /// Stops the sound.
         /// </summary>
         /// <remarks>This will set the time position back to the beginning.</remarks>
-        public void Stop() => this.sound.Stop();
+        public void Stop()
+        {
+            if (this.sound.Unloaded)
```

Comment:
NOTE: This was a bug fix here.
This was stopping the sound when it was already unloaded which was causing an exception within the CASL library.